src/main/java/com/fortanix/sdkms/performance/helper/EncryptionDecryptionHelper.java

```diff
@@ -29,7 +29,7 @@ public EncryptRequestEx createEncryptRequest(SobjectDescriptor key, String plain
         return new EncryptRequestEx().key(key).alg(this.algorithm).plain(plainText.getBytes()).mode(this.mode).iv(this.iv).ad(this.ad).tagLen(this.tagLen);
     }
 
-    public DecryptRequestEx createDecryptRequest(SobjectDescriptor key, byte[] cipher) {
-        return new DecryptRequestEx().key(key).alg(this.algorithm).cipher(cipher).mode(this.mode).iv(this.iv).ad(this.ad);
+    public DecryptRequestEx createDecryptRequest(SobjectDescriptor key, byte[] cipher, byte[] tag) {
```

Comment:
Update this method to take IV also as the input. Then you can use this directly from AESHelper.java(line 71)